### DISCUSSION TITLE
Heating R1233ZDE from sub-cooled liquid to super-heated vapor: discrepancies in enthalpy drop in comparison to well known commercial software

### DISCUSSION THREAD

**Discussion by leopoldomar** [usnistgov/REFPROP-issues#678] [Q&A]
Dear all,

I'm developing a Fortran code for geothermal plants calculation.

For testing and validation purpose I've simulated the heating of R1233ZDE, from inlet condition of 50 C and 26.27 bar to 170 C and 25.88 bar, through an economizer, an evaporator and a superheater which are part of the components library of the code. The fluid I'm exchanging heat with is a mixture of H2O, CO2 and H2S. 

To compute the thermodynamic properties of R1233ZDE I'm directly calling the following REFPROP Fortran subroutines:

- for calculating H, known P and T
    1. tpflsh for single-phase condition
    2. pqflsh for two-phase condition

-  for calculating T, known P and H
    1. phflsh always

The issue I've found is that the results I obtain in terms of enthalpies of the R1233 fluid are slightly different from the ones I get from a commercial software I'm using as reference for validating the code.

Specifically, the DH or enthalpy drop I compute is 2248 kJ/kg instead of 2364 I see from the commercial code. The difference is just 5%, but it is affecting the outlet condition of the mixture used to heat R1233 by 26%!
On H2O, CO2 and H2S side, the target outlet condition is 80 C, while I compute 106 C, when the inlet T is 180 C imposed (26 C difference over a 100 C expected DT is 26%)

Forcing the DH on R1233 side to be 2364 kJ/kg, thus increasing the outlet T, the resulting outlet T on mixture side is 80 C, so on this side it seems REFPROP is aligned with the commercial software. I must be mistaking something when recalling the REFPROP subroutines to calculate R1233 properties, but I really don't know what.

Do you have any idea or suggestion?

Thanks and regards!

Leopoldo

**Comment by ianhbell** [comment]
I guess this is simply a question of reference states. Make sure that your "commercial software" and REFPROP are using the same reference state and you should be golden.

**Reply by leopoldomar** [reply]
Hi Ian, thanks for your reply, anyway I have some doubts about your suggestion; the reference state affects the enthalpy value for the single state points, not the difference between two different state points, that is exactly what I pointed out. I see small differences in the delta enthalpy from an initial state and a final, comparing to Hysys (that's the commercial software, nothing to hide :-) about it).

However I can say without a doubt that the reference state are different, as the single state points have totally different enthalpies from REFPROP to Hysys; just to check, I can try a way to change the reference state to match the single state points enthalpies, but I suspect this won't change my results with the delta enthalpies.

**Comment by ianhbell** [comment]
Are you sure they use the same EOS?

**Reply by leopoldomar** [reply]
That's a good point; from Hysys I see that designers have used Peng-Robinson model for R1233 and Extended NRTL for the mixture H2O, CO2 and H2S.

Actually I've found in the documentation the "flags" subroutine where you can specify some flags with attributes..."PR" should be the for Peng-Robinson, and possible options are 0,2,3. I've tried all but results don't change.
Also I see there's a "setmod" subroutine where you can set a lot of different features, but I didn't play with it yet.

**Reply by ianhbell** [reply]
That is a VERY different EOS. REFPROP is definitely more accurate if the comparison is against PR
